osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -79,7 +96,7 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
                 );
 
             double starRating = basePerformance > 0.00001
-                ? Math.Cbrt(OsuPerformanceCalculator.PERFORMANCE_BASE_MULTIPLIER) * 0.027 * (Math.Cbrt(100000 / Math.Pow(2, 1 / 1.1) * basePerformance) + 4)
+                ? Math.Cbrt(OsuPerformanceCalculator.PERFORMANCE_BASE_MULTIPLIER) * 0.026 * (Math.Cbrt(100000 / Math.Pow(2, 1 / 1.1) * basePerformance) + 4)
```

Comment:
Why?

Reply:
> Why?

Pretty sure this was changed because adding length bonus juiced up some maps way too much, kermit can probably elaborate better

Reply:
puttng length bonus into star rating increases star rating for everything, so I decreased multiplier to account for that and keep the average about the same

Reply:
Was changing `PERFORMANCE_BASE_MULTIPLIER` not good enough? Ideally we avoid touching the actual formula if we can help it.

Reply:
changing `PERFORMANCE_BASE_MULTIPLIER` would affect final pp values, when the problem is with the star rating => pp relationship. Including length in star rating means that for the same pp value star rating will be higher, and so the multiplier in the formula is decreased to keep it on average the same.